### PR TITLE
gjs: add autoconf@213 as resource

### PR DESCRIPTION
**Pull request by tschoonj**
`autoconf@213` has been [marked for deletion](https://github.com/Homebrew/homebrew-core/pull/52141), but since `gjs` needs it to build its resource SpiderMonkey, it needs to be added as a resource as well.

@bo98 @smillerdev

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by Bo98** [comment]
Given it's usefulness for users who may be interacting with Mozilla components, and its high install count, I'm fine with the formula staying around. But I do realise that would be an exception given it's 20 years old.

I invite other @Homebrew/maintainers to give comment.

**Comment by MikeMcQuaid** [comment]
I'm also happy with it staying around. It's still needed to build modern software.

**Comment by iMichka** [comment]
Yeah, I agree that vendoring is a good solution for this case.

**Review by chenrui333 — DISMISSED** [review]

**Comment by chenrui333** [comment]
LGTM, 👍

**Comment by tschoonj** [comment]
So close this PR or not? There's no harm in merging it in I think, even if at some point later we decide to remove `autoconf@213`

**Comment by Bo98** [comment]
Seems to be tie in terms of votes at the moment.

**Comment by MikeMcQuaid** [comment]
A compromise would be to leave it until https://github.com/Homebrew/brew/issues/7029 then immediately deprecate it. We could start building a list in that issue of stuff that needs deprecation

**Comment by SMillerDev** [comment]
I think vendoring this but keeping the formula around until the depreciation feature is a good compromise.

**Comment by MikeMcQuaid** [comment]
I'm not sure I understand why vendoring and keeping a formula around are a good idea? 

What problem does this solve for our users or maintainers?

**Comment by SMillerDev** [comment]
Since the work here is already done, it would allow us to deprecate the formula more easily in the future iyam.

**Comment by MikeMcQuaid** [comment]
Cool. I'd be happy to see this stay open until then then and merged once we have the deprecation stuff?

**Comment by SMillerDev** [comment]
Works for me, there's plenty of other PRs I can look at to get the number down.

**Comment by MikeMcQuaid** [comment]
We have a 2.2.12 tag now so we can use the deprecation stuff.

**Comment by tschoonj** [comment]
Ok, I will rebase this one against master

**Comment by tschoonj** [comment]
I expect that the build will now be fixed.

By the way, it is perhaps best to keep `autoconf@213` around a bit longer: the [Mozilla build instructions for macOS](https://firefox-source-docs.mozilla.org/setup/macos_build.html#a-install-dependencies-via-homebrew) rely on this formula 😄

**Comment by MikeMcQuaid** [comment]
> By the way, it is perhaps best to keep `autoconf@213` around a bit longer: the [Mozilla build instructions for macOS](https://firefox-source-docs.mozilla.org/setup/macos_build.html#a-install-dependencies-via-homebrew) rely on this formula 😄

I'm game for keeping it around then 👍

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**
